### PR TITLE
[INFRA-660] Dockerized cloudlift 

### PR DESCRIPTION
**Pull request by siliconsenthil**
https://rippling.atlassian.net/browse/INFRA-660

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+                sh '''
+                    git remote set-url origin git@github.com:Rippling/cloudlift || git remote add origin git@github.com:Rippling/cloudlift
+                    if ! [ -z "${COMMIT_ID}" ]; then
+                        echo "Checking out custom commit id: ${COMMIT_ID}"
+                        git checkout ${COMMIT_ID}
+                    fi
+                    git fetch --prune origin "+refs/tags/*:refs/tags/*"
+                    echo "Tagging this commit: $(git rev-parse HEAD)"
+                    git tag ${TAG}
+                    git push --tags
```
Comment (aswinkarthik) [inline Jenkinsfile:19]: can we push the tag we create? `push --tags` is to push all tags. Even though there is only one tag to push, i feel it is better to just explicitely just push our tag. What is your opinion?

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+                sh '''
+                    docker build -t cloudlift:${TAG} .
+                '''
+            }
+        }
+        
+        stage('Push to ECR') {
+            steps {
+                sh '''
+                    aws ecr get-login-password --region us-west-2 | docker login --username AWS --password-stdin 408750594584.dkr.ecr.us-west-2.amazonaws.com
```
Comment (aswinkarthik) [inline Jenkinsfile:36]: Shall we configure REGION and ACCOUNT_ID in jenkins as well? We can just use them as env variable here?

**Review by aswinkarthik — CHANGES_REQUESTED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (bkdonline) [reply Jenkinsfile:19]: Agreed. Fixed.

File: Jenkinsfile (hunk 2 above)
Comment (bkdonline) [reply Jenkinsfile:36]: image
Makes sense. Fixed.

**Review by aswinkarthik — APPROVED** [review]

**Pull request merged**